tests/foreman/ui/test_myaccount.py

```diff
@@ -67,38 +67,13 @@ def tearDown(self):
         super(MyAccountTestCase, self).tearDown()
         self.account_user.delete()
 
-    def logged_test_user(self):
+    def session_test_user(self):
```

Comment:
Because I can not mess with the the default user created for Session, once it can lead to errors on another tests. This method is creating a session with the user created on setUp. So each test has an isolated user to play with.